src/app/app.component.scss

```diff
@@ -2,4 +2,9 @@
   overflow: hidden;
   height: 100vh;
   width: 100vw;
-}
\ No newline at end of file
+}
+
+::ng-deep ngx-feedback-main-btn {
```

Comment:
I just know that Jon would have had something to say about the use of ng-deep..... looks to be deprecated https://blog.angular-university.io/angular-host-context/